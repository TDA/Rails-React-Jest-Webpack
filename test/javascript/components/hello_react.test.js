import Hello from 'hello_react';

test('render a Hello component', () => {
    const wrapper = shallow(
        <Hello name='saisai'/>
    );
    expect(wrapper).toMatchSnapshot();
});