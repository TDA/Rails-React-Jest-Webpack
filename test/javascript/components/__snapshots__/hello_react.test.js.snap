// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a Hello component 1`] = `
<div>
  Hello I am from Webpacker! 
  saisai
  !
</div>
`;
